package-lock.json

```diff
@@ -3625,16 +3739,31 @@
       "requires": {
         "@bbc/gel-foundations": "^3.4.3",
         "@bbc/psammead-styles": "^4.2.0"
+      },
+      "dependencies": {
+        "@bbc/gel-foundations": {
+          "version": "3.4.3",
+          "resolved": "https://registry.npmjs.org/@bbc/gel-foundations/-/gel-foundations-3.4.3.tgz",
+          "integrity": "sha512-rOPuintdkftYDE0GcY8B5K/AR5tLAMv7E+jD0plk32aEBojTXgttSYW9Da4NBKxLw69N9Px29YEu1CU0is1j9w=="
+        }
       }
     },
     "@bbc/psammead-storybook-helpers": {
-      "version": "8.2.3",
-      "resolved": "https://registry.npmjs.org/@bbc/psammead-storybook-helpers/-/psammead-storybook-helpers-8.2.3.tgz",
-      "integrity": "sha512-9mm3MqDjIgUlMnAX2urLAfTdVxAFYeIQFLA6Hwbe3uqUZBK0IWFInqftkbLeb4rAtWUquN89ngGDYP53kAw9SA==",
+      "version": "8.2.5",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-storybook-helpers/-/psammead-storybook-helpers-8.2.5.tgz",
+      "integrity": "sha512-o0Zi8j7sIW8CdCtE/AZxQFwZsimCmuzEsUUeXX85UZlLaJ9uN+AtD4gOjcEQycCbT8YzQBEKR/JrEnEwweXZAA==",
       "dev": true,
       "requires": {
-        "@bbc/gel-foundations": "^3.4.3",
+        "@bbc/gel-foundations": "^4.0.1",
         "react-helmet": "^5.2.1"
+      },
+      "dependencies": {
```

Comment:
You shouldn't need all of these dependencies blocks if the version for this package is the same as the simorgh version! Try removing all of these for when the version is 4.0.1, it should work and you should see a big reduction in bundle size :)